src/PySXO/workflow.py

```diff
@@ -1,8 +1,51 @@
-from .core.base import Base
+import json
+
 from attrdict import AttrDict
+from typing import Union, Dict, List
+
+from .core.base import Base
 
 class Workflow(Base):
+    def __init__(self, sxo, raw=None):
+        super().__init__(sxo, raw=raw)
+
     def __getattr__(self, key):
         if isinstance(self._json.get(key), dict):
             return AttrDict(self._json[key])
-        return self._json.get(key)
\ No newline at end of file
+        return self._json.get(key)
+
+    @property
+    def id(self) -> str:
+        return self._json.id
+
+    @property
+    def start_config(self) -> AttrDict[Union[Dict, List]]:
+        return AttrDict(self._sxo._get(url=f'/api/v1/workflows/ui/start_config?workflow_id={self.id}'))
+
+    def start(self, **kwargs) -> Union[List, Dict]:
+        body = {"input_variables":[]}
+        for variable_id, variable_definition in self.start_config.property_schema.properties.items():
+            if variable_definition["title"] in kwargs.keys():
+                if isinstance(kwargs[variable_definition["title"]], dict):
+                    value = json.dumps(kwargs[variable_definition["title"]])
+                else:
+                    value = kwargs[variable_definition["title"]]
```

Comment:
Can You explain this? Is it possible that SXO api returns title as dict?

Reply:
This was actually your idea of solving the problem with SXO not receiving dicts LMAO

We are trying to dump everything into string repr so it is readable by SXO.